### PR TITLE
Bugfix/protect override

### LINKED ISSUE #95: Docs can be overwritten
One should not be able to overwrite docs

### PR DESCRIPTION
**Pull request by fliiiix**
fixes #95

### CODE REVIEW

**Comment by fliiiix** [comment]
needs https://github.com/randombenj/docat/pull/105

File: docat/app.py [hunk 1]
```diff
@@ -38,10 +37,18 @@ def upload(project, version):
     base_path = project_base_path / version
     target_file = base_path / secure_filename(uploaded_file.filename)
 
+    if base_path.is_dir():
```
Comment (randombenj) [inline docat/app.py:40]: So symlinks (tags) can still be overwritten by new versions?
Is this intended behaviour?

**Review by randombenj — APPROVED** [review]

File: docat/app.py (hunk 1 above)
Comment (fliiiix) [reply docat/app.py:40]: good question, you are right currently this implementation allows to overwrite a tag with a actual thing maybe this should be verboten

Comment (fliiiix) [reply docat/app.py:40]: i will fix that

Comment (fliiiix) [reply docat/app.py:40]: turns out `is_dir` is true for a symlink which point to a dir but i will change it to `exists` regardless

Comment (fliiiix) [reply docat/app.py]: and i added a test ;)

**Pull request merged**